ftw/subsite/browser/ressources/integration.ftw-mobile.theme.scss

```diff
@@ -1,4 +1,11 @@
-/* This file is only used when ftw.theming and ftw.mobile is installed. */
+// This file is only used when ftw.theming and ftw.mobile is installed.
+
+// Redefine used ftw.mobile scss variables.
+// This prevents us from sass compilation error if ftw.subsite is installed and ftw.mobile only in the PATH
+$size-mobile-button: 72px !default;
+$font-size-mobile-button: 2em !default;
```

Comment:
@raphael-s If I understand it correctly, those variable definitions are no longer necessary at all, the ZCML-change alone resolves the problem: the `integration.ftw-mobile.theme.scss` is not loaded at all when `ftw.mobile` is available but not installed.
It does not make sense to load `integration.ftw-mobile.theme.scss` without `ftw.mobile`, therefore those variable redifinitions do not make sense.